App.js

```diff
@@ -1,21 +1,68 @@
 import React from 'react';
```

Comment:
This file looks really solid but this code shouldn't reside in `App.js`. Instead, for each screen, there should be a corresponding file. So you should create a `Search.js` file.